src/yashCJun20/Assignment22/VerifyPalindromeNumber.java

```diff
@@ -0,0 +1,25 @@
+package yashCJun20.Assignment22;
+class VerifyPalindromeNumber{
+	void toVerifyPalindromeNumberOrNot(int num){
+		int remender;
+		int reverseNumber=0;
+		int originalNumber;
+		originalNumber=num;
+		while (num!=0)
+			{
+				remender=num%10;
+				reverseNumber=reverseNumber*10+remender;
+				num=num/10;
```

Comment:
reverseNumber=(reverseNumber*10) + (num%10);
num=num/10;

Use proper brackets when you are performing multiple operations.